frontend/src/metabase/timelines/questions/components/EventCard/EventCard.tsx

```diff
@@ -56,6 +71,8 @@ const EventCard = ({
       isSelected={isSelected}
       onClick={handleEventClick}
     >
+      <Checkbox checked={!isHidden} onClick={handleChangeVisibility} />
```

Comment:
Ideally should be `onChange`, not `onClick`, but then the click propagates and toggles select. Which can be treated by reusing `handleAsideClick` but then it demands changing that function's name and things start getting polluted.